data/ui/main_window.ui

```diff
@@ -172,6 +174,7 @@
                     </child>
                     <child>
                       <object class="GtkButton" id="back_button">
+                        <property name="visible" bind-source="leaflet" bind-property="folded" bind-flags="sync-create"/>
                         <property name="visible">1</property>
```

Comment:
You can remove this line otherwise the binding won't work.

Reply:
Ah, forgot to remove that one